pages/workflow/overview.mdx

```diff
@@ -0,0 +1,50 @@
+---
+layout: 'guide'
+title: 'Avo Workflow'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+
+import { FunctionComponent } from 'react';
+import Zoom from 'react-medium-image-zoom';
+
+The Avo workflow is the recommended approach for a team working on an analytics release. The workflow is helpful both when new analytics tracking is being implemented, as well as when existing tracking is being updated.
+
+The Avo workflow for analytics releases is inspired by the general best practices of software development using branched change management, peer-reviews, and approval workflows.
+
+1. **Plan** analytics update
+2. **Review** suggestions
+3. **Request** implementation instructions with developers
+4. **Implement** analytics events
+5. **Validate** implementation
+6. **Merge** Avo branch and **Publish** schema to downstream schema registries
+
+<Zoom>
+  <center>
+    <img
+          width="100%"
+          src={require('../../images/workflow/overview/avo-workflow.png')}
+          alt="The Avo Workflow: Plan, review, implement, validate and publish your tracking changes"
+    />
+  </center>
+</Zoom>
```

Comment:
It looks like there is something funky going on here, I can't click the image, but there is this tiny empty button right below it which expands it, but it expands in a weird place.

<img width="752" alt="Screenshot 2022-07-21 at 22 12 02" src="https://user-images.githubusercontent.com/3513968/180323827-f60829f3-ead0-423d-ab8b-b72bc5e08329.png">

If we can't easily fix I suggest we ship without the zoom

Reply:
Okay, this is super weird, when I run this on my localhost the zoom works perfectly and this tiny box at the bottom is not there. I don't know why this appears like this now.